File: app/src/main/java/centurion/dev/browniepoints/DataModel/PointsAccount.java
package centurion.dev.browniepoints.DataModel;

public class PointsAccount {

    Long id;

    int points;

    String name;

    public void setId (Long id) {
        this.id = id;
    }

    public Long getId () {
        return id;
    }

    public void setPoints (int points) {
        this.points = points;
    }

    public int getPoints () {
        return points;
    }

    public void setName (String name) {
        this.name = name;
    }

    public String getName () {
        return name;
    }

}

File: app/src/main/java/centurion/dev/browniepoints/DataModel/PointsAccount.kt
package centurion.dev.browniepoints.DataModel

class PointsAccount(var id: Long?, var points: Int, var name: String) : Comparable<PointsAccount> {

    override fun compareTo(pointsAccount: PointsAccount): Int {
        val compareID = pointsAccount.id!!
        val delta = this.id!! - compareID
        return delta.toInt()
    }

}
